content/client-server-api/modules/end_to_end_encryption.md

```diff
@@ -1257,10 +1257,10 @@ the following format:
 
 `AuthData`
 
-| Parameter  | Type       | Description                                                                                      |
-| -----------| -----------|--------------------------------------------------------------------------------------------------|
-| public_key | string     | **Required.** The curve25519 public key used to encrypt the backups, encoded in unpadded base64. |
-| signatures | Signatures | Optional. Signatures of the ``auth_data``, as Signed JSON                                        |
+| Parameter  | Type       | Description                                                                                                           |
```

Comment:
we have ways of generating these tables which we should start using for the crypto section too. See https://github.com/matrix-org/matrix-spec/pull/1062 for how we described `m.relates_to`, for example.

Reply:
Unfortunately, I don't have time to figure this one out right now. Mind if we postpone it to a later PR?

Reply:
Preferably not, as otherwise it won't ever get fixed.

Reply:
I'd prefer to convert the tables too, but really can't afford to fiddle with this now. But the changes are useful on their own and unrelated to the table generation method, no?

Reply:
The changes might be useful, but we really can't afford to keep putting off the tech debt.

Reply:
Yeah, if you do get around to it before me, feel free to do it. Thanks!

Reply:
ok, I have made these changes in #1166. Unfortunately it is going to introduce conflicts with this PR which will now need to be resolved.